### PR TITLE
[docs] Explain how to correctly map Expo published source maps to Sentry Issues

### PR DESCRIPTION
**Pull request by aldegoeij**
# Why

Clarification of Sentry documentation, how to map Sentry Issues to the correct Sentry Release source maps as published by Expo

# How

doc update

# Test Plan

n.a.

### CODE REVIEW

File: docs/pages/versions/unversioned/guides/using-sentry.md [hunk 1]
```diff
@@ -79,6 +79,19 @@ The correct `authToken` value can be generated from the [Sentry API page ](https
 
 With the `postPublish` hook in place, now all you need to do is hit publish and the sourcemaps will be uploaded automatically. We automatically assign a unique release version for Sentry each time you hit publish, based on the version you specify in `app.json` and a release id on our backend -- this means that if you forget to update the version but hit publish, you will still get a unique Sentry release. If you're not familiar with publishing on Expo, you can [read more about it here](https://blog.expo.io/publishing-on-exponent-790493660d24).
 
+In order to use the published release sourcemaps with Issues in Sentry, you'll have to set your Expo `revisionId` as Sentry release identifier, like so:
+
+```javascript
+// e.g. in your `App.js`
+import Constants from "expo-constants"
```
Comment (ide) [inline docs/pages/versions/unversioned/guides/using-sentry.md]:
```suggestion
import Constants from 'expo-constants';
```

File: docs/pages/versions/unversioned/guides/using-sentry.md [hunk 2]
```diff
@@ -79,6 +79,19 @@ The correct `authToken` value can be generated from the [Sentry API page ](https
...
+
+// ... initialize Sentry first
+
+Sentry.setRelease(Constants.manifest.revisionId)
```
Comment (ide) [inline docs/pages/versions/unversioned/guides/using-sentry.md]:
```suggestion
Sentry.setRelease(Constants.manifest.revisionId);
```

File: docs/pages/versions/unversioned/guides/using-sentry.md [hunk 3]
```diff
@@ -79,6 +79,19 @@ The correct `authToken` value can be generated from the [Sentry API page ](https
 
 With the `postPublish` hook in place, now all you need to do is hit publish and the sourcemaps will be uploaded automatically. We automatically assign a unique release version for Sentry each time you hit publish, based on the version you specify in `app.json` and a release id on our backend -- this means that if you forget to update the version but hit publish, you will still get a unique Sentry release. If you're not familiar with publishing on Expo, you can [read more about it here](https://blog.expo.io/publishing-on-exponent-790493660d24).
 
+In order to use the published release sourcemaps with Issues in Sentry, you'll have to set your Expo `revisionId` as Sentry release identifier, like so:
```
Comment (ide) [inline docs/pages/versions/unversioned/guides/using-sentry.md]:
```suggestion
In order to use the published release source maps with Issues in Sentry, you'll have to set your Expo `revisionId` as the Sentry release identifier, like so:
```

**Review by ide — CHANGES_REQUESTED** [review]

**Comment by aldegoeij** [comment]
All accepted!

**Comment by ide** [comment]
Please push the changes so that we can land them.

**Pull request merged**